README.md

```diff
@@ -1 +1,52 @@
-# LeaderBoard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# LeaderBoard
+
+> I build an LeaderBoard App using HTML, CSS(flexbox), Javascript, and Webpack, by also keeping good practice with the Linters correction.
+> I create the form input fields, where user can add any name and score of there choice.
+> I created the desktop version using media queries.
+
+![screenshot](./images/screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML and CSS
+- JavaScript
+- local storage
+- Technologies: Webpack and GitHub
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+* VS Code - A studio code editor.
+* Node Package Manage: For installing and managing packages used in the project.
+* Linters setup
+* GitHub - For storing our codes and managing the git repo.
+
+### Setup
+* Create a Repo 
+  * Go to github and create a repo
+  * Copy the ssh code to clone
+* Open the Commandline
+  * Navigate To the right directory
+  * Run ```Git Clone https://github.com/idungstanley/LeaderBoard.git```
+  * Run ```code .``` to open your VScode Editor
+* Create the Github\workflows Folder
+  * Create a Linters.yml file and configure
+
+## Live Demo
+
+[Site is live. Visit now!](https://idungstanley.github.io/LeaderBoard/)
```

Comment:
- [ ] The live demo link is broken. I recommend fixing it or removing it if the live demo link is unavailable. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.